igvm/templates/user_data.cfg

```diff
@@ -0,0 +1,92 @@
+#cloud-config
```

Comment:
user_data.cfg seems a bit generic to me.

Reply:
In AWS console its called `user_data` and the API parameter is `UserData`. Should I prefix it with `aws_` ?

Reply:
Or maybe `aws_cloudinit.cfg` ?

Reply:
renamed it to `aws_user_data.cfg`